wiring/inc/spark_wiring_posix_common.h

```diff
@@ -73,8 +78,58 @@ inline void ipAddressPortToSockaddr(const IPAddress& addr, uint16_t port, struct
 #endif // HAL_IPv6
 }
 
+inline uint8_t netmaskToPrefixLength(struct sockaddr* saddr) {
+    if (saddr->sa_family == AF_INET) {
```

Comment:
Add `CHECK_TRUE(saddr, SYSTEM_ERROR_INVALID_ARGUMENT);`

Reply:
Functions here are helpers so the requirement is that saddr is valid.